files/image_config/constants/constants.yml

```diff
@@ -30,10 +30,17 @@ constants:
           - "deny 0::/0 le 59"
           - "deny 0::/0 ge 65"
           - "permit fe80::/64"
+    bbr:
+      enabled: true
     peers:
       general: # peer_type
         db_table: "BGP_NEIGHBOR"
         template_dir: "general"
+        bbr:
+          PEER_V4:
+            - ipv4
+          PEER_V6:
+            - ipv6
```

Comment:
why not use device metadata role to identify if a peer is a tor or not?

Reply:
1. I don't have neighbor metadata in public. That was disabled by multiple requests from the community. Without this change I can implement the change in internal, but not in the public. But we want to test the feature on public too.
2.  I need a way to define, what address families are used by each peer-group for BBR.

Reply:
I think this adds additional maintaining cost. it is better to check if neighbor metadata is available or not, if it is available, then use it. otherwise, look for bbr definition. do you agree?

Reply:
I don't agree here, sorry
1. We need to define mapping between address families and peer-groups. We can't extract this information from the minigraph, it is available only in templates.
2. Currently bgpcfgd doesn't know anything about the device neighbor metadata, also as T0, T1, T2 and so on. That was requested explicitly by community. So I add an option to depend on the neighbor metadata presence, but I enable that option only for internal image. All other operations with T0, T1 and so on is done inside of the internal templates.
3. We need to test this functionality using the public image, so we need a way to define this without neighbor meta
4. Current approach allows us to have T0 peer-groups, which doesn't support BBR.
5. My goal is to make templates downloadable from external source, like we download acl rules. I think of having a tar archive with templates + metadata file in the archive describing the metadata of the templates.